pyteal/stack_frame.py

```diff
@@ -365,9 +407,16 @@ def _walk_asts(
                     walker_args = [expr.value]
                     if expr.index_expression:
                         walker_args.append(expr.index_expression)
+                case SubroutineCall():
+                    subdef = expr.subroutine.get_declaration_by_option(False)
+                    if subdef:
+                        walker_args.append(subdef)
+                    subdef = expr.subroutine.get_declaration_by_option(True)
+                    if subdef:
+                        walker_args.append(subdef)
```

Comment:
@ahangsu - introducing these lines of code into the sourcemapper, actually changed the resulting teal. I'm trying to understand why.

Reply:
seems fixed? For we introduced a fix #684 and the problem should go away

Reply:
Absolutely. Please disregard this stale comment. Also, I'm about to close this draft in favor of #689